.github/workflows/ci.yml

```
@@ -33,9 +33,27 @@ jobs:
+      run: |
+        #docker login --username ${{ github.actor }} --password ${{ secrets.GITHUB_PAT }} ghcr.io 
+        docker login --username ${{ github.actor }} --password ${{ secrets.GITHUB_TOKEN }} ghcr.io  
+        docker build . --tag ghcr.io/lecaqquochung/ubuntu-latest:latest
+        docker push ghcr.io/lecaoquochung/ubuntu-latest:latest
     - name: ubuntu-lastest
-      run: docker build . --file ./ubuntu/latest.Dockerfile --tag "ubuntu-latest:$(date +%s)"
-  
+      run: |
+        # docker build . --file ./ubuntu/latest.Dockerfile --tag "ubuntu-latest:$(date +%Y-%m-%d_%H:%M:%S)"
+        # docker login --username lecaoquochung --password ${{ secrets.GITHUB_TOKEN }} ghcr.io
+        docker login --username ${{ github.actor }} --password ${{ secrets.GITHUB_TOKEN }} ghcr.io  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)